src/main/java/com/epam/lab/paymentsystem/service/OperationService.java

```diff
@@ -1,7 +1,48 @@
 package com.epam.lab.paymentsystem.service;
 
 import com.epam.lab.paymentsystem.entities.Operation;
+import java.util.List;
 
+/**
+ * OperationService is the interface that specifies the service methods.
+ */
 public interface OperationService {
+
+  /**
+   * History operation.
+   *
+   * @return operation list.
+   */
+  List<Operation> historyOperation();
+
+  /**
+   * History operation by account.
+   * * @param accountId long.
+   *
+   * @return operation list
+   */
+  List<Operation> historyAccountOperation(long accountId);
```

Comment:
Please, fix naming. For example, as above

Reply:
Do you mean to make a @Override method?